xml/System.Security.Principal/WindowsImpersonationContext.xml

```diff
@@ -240,9 +237,6 @@ Application code does not call this method; it is automatically invoked during g
  ]]></format>
         </remarks>
         <exception cref="T:System.Security.SecurityException">An attempt is made to use this method for any purpose other than to revert identity to self.</exception>
-        <block subset="none" type="overrides">
```

Comment:
@gewarren this is the first time I see a `block` tag. I wanted to see how it rendered in the [MS Docs page for this API](https://docs.microsoft.com/en-us/dotnet/api/system.security.principal.windowsimpersonationcontext.undo?view=netframework-4.8), but that tag is not showing up at all. Do you know what it was being used for?

Reply:
Oh, it _is_ showing up! But it's completely separated from the exception where it was added. It's in its own section called "Notes to Inheritors". That's a really cool feature to be aware of:

![image](https://user-images.githubusercontent.com/1175054/112202746-f54e3800-8bce-11eb-9e04-a9ab9be1abaf.png)

Reply:
Yes, I investigated the same thing myself yesterday as I hadn't seen the syntax before. There's also `<block subset="none" type="usage">`, which generates a "Notes to Callers" section at the end of the page.